alf/utils/dist_utils.py

```diff
@@ -157,3 +157,23 @@ def calc_default_target_entropy(spec):
         q = 1 - p
         e = -p * np.log(p) - q * np.log(q)
     return e * dims
+
+
+def calc_default_max_entropy(spec):
+    """Calc default max entropy
+    Args:
+        spec (TensorSpec): action spec
+    Returns:
+        A default max entropy for adjusting the entropy weight
+    """
+    if tensor_spec.is_continuous(spec):
+        # pick a very conservative high value in the continuous case
+        # (upper bound for free; better than nothing)
+        e = 100
```

Comment:
This is too  big.  The entropy for Gaussian is 0.5*log(2*pi*e*sigma^2). Even if sigma=10, it's just 3.72. So I would say 2 per dimension might be reasonable.